fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticStorageInitializer.java

```diff
@@ -57,7 +58,8 @@ public void run() {
         }
         while (true) {
             try {
-                Thread.currentThread().join(Config.statistics_table_creation_retry_interval_in_seconds * 1000L);
+                Thread.currentThread()
+                        .join(StatisticConstants.STATISTICS_TABLE_CREATION_RETRY_INTERVAL_IN_SECONDS * 1000L);
                 createDB();
```

Comment:
@morningman do we have any other internal table need to save in future? if the anwser is yes, i think we should
1. rename the database to a common name
2. mv this class to another place and rename it